docs/core/tools/dotnet-pack.md

```diff
@@ -53,91 +51,59 @@ Web projects aren't packable by default. To override the default behavior, add t
 
 ## Arguments
```

Comment:
@livarcocc Can you or someone on your team review this section for the versions that things first appeared in, if it wasn't 1. Some are missing or look wrong to me.

Reply:
Oh I've missed some of the x.x that I left as placeholders. But yes, reviewing for accuracy would be nice.

Reply:
I've fixed the missing version and also double checked the changes I've made for the 2.0 SDK release [here](https://github.com/dotnet/docs/commit/81b08d3c9acf0b3534047910aa8ff9bf8559940d#diff-8071d44406b4c597993915f7deca9fbb). It seems those options were introduced for 2.0.

I can't download the 2.0 SDK installer any longer and I don't have it on my machine.